Source/Dafny/DafnyMain.cs

```diff
@@ -231,9 +232,12 @@ private class IncludeComparer : IComparer<Include> {
       return null; // Success
     }
 
-    public static bool BoogieOnce(ExecutionEngine engine, string baseFile, string moduleName, Microsoft.Boogie.Program boogieProgram, string programId,
-      out PipelineStatistics stats, out PipelineOutcome oc) {
-      programId = (programId ?? "main_program_id") + "_" + moduleName;
+    public static async Task<(bool IsVerified, PipelineOutcome Outcome, PipelineStatistics Statistics)> BoogieOnce(
+      TextWriter output,
+      ExecutionEngine engine, string baseFile,
+      string moduleName,
+      Microsoft.Boogie.Program boogieProgram, string programId) {
```

Comment:
Curly on its own line

Reply:
I'll change it, but why?

Reply:
The formatter puts it on the same line

Reply:
Let's stick with dotnet-format then.

Reply:
For readability: `var moduleId` below ends up on the same column as `Microsot.Boogie`.  Let's stick with dotnet-format though, sorry for the noise.